ublue/kcm_ublue/kcm_ublue.spec

```diff
@@ -0,0 +1,58 @@
+Name:           kcm_ublue
+Version:        0.5.2
+Release:        1%{?dist}
+Summary:        KCM for KDE-based Universal Blue images
+
+URL:            https://github.com/ledif/kcm_ublue
+Source:         https://github.com/ledif/kcm_ublue/archive/refs/tags/v0.5.2.zip
+License:        Apache-2.0
+
+BuildRequires:  cmake
+BuildRequires:  gcc-g++
+BuildRequires:  extra-cmake-modules
+BuildRequires:  kf6-kauth-devel
+BuildRequires:  kf6-kcmutils-devel
+BuildRequires:  kf6-kcodecs-devel
+BuildRequires:  kf6-kcolorscheme-devel
+BuildRequires:  kf6-kconfig-devel
+BuildRequires:  kf6-kconfigwidgets-devel
+BuildRequires:  kf6-kcoreaddons-devel
+BuildRequires:  kf6-ki18n-devel
+BuildRequires:  kf6-kservice-devel
+BuildRequires:  kf6-kwidgetsaddons-devel
+BuildRequires:  qt6-doc-devel
+BuildRequires:  qt6-qtbase-devel
+BuildRequires:  qt6-qtdeclarative-devel
+BuildRequires:  qt6-qttools-devel
+
+%description
+KDE Configuration Module (KCM) for Aurora.
```

Comment:
There doesn't seem to be any aurora-specific configuration in this file.  
Could this description be changed to "KDE Configuration Module (KCM) for Ublue"?

Reply:
Ohh, the Aurora-specific configuration is in your repo!  This can be ignored